### PR TITLE
Updates NEWS for the 1.1.1 release

### PR DESCRIPTION
**Pull request by mattcaswell**

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -8,19 +8,36 @@
   Major changes between OpenSSL 1.1.0i and OpenSSL 1.1.1 [in pre-release]
 
       o Support for TLSv1.3 added
```
Comment (mspncp) [inline NEWS]: Only one line for the largest new feature? How modest!

**Comment by mattcaswell** [comment]
Pushed.

File: NEWS (hunk 1 above)
Comment (kaduk) [reply NEWS]: One could perhaps talk about the state machine rewrite, extension handling rewrite, and packet-construction code that were added to support it.

Comment (mattcaswell) [reply NEWS]: The state machine rewrite and packet construction code were done for 1.1.0. There were of course new states added for TLSv1.3. The packet *writing* code is new in 1.1.1

Comment (mattcaswell) [reply NEWS]: Of course I realise what I just wrote about packets made absolutely no sense... :-)

Comment (mspncp) [reply NEWS]: @mattcaswell, how about stealing some lines from your [blog post](https://www.openssl.org/blog/blog/2018/02/08/tlsv1.3) or mentioning [RFC8446](https://tools.ietf.org/html/rfc8446) :wink: ?

Comment (mattcaswell) [reply NEWS]: See #7167

**Pull request closed without merging**